examples/control-elements/media-captions-button.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width">
+  <script type="module" src="../../dist/media-captions-button.js"></script>
+</head>
+<body>
+
+<h1>&lt;media-captions-button&gt;</h1>
+
+<h3>Default (no captions or subtitles)</h3>
+<media-captions-button></media-captions-button>
+
+<h3>Closed Captions on ("captions showing")</h3>
+<media-captions-button media-captions-showing="en:English%20Closed%20Captions" media-captions-list="en:English%20Closed%20Captions"></media-captions-button>
+<!-- 
+<h3>Closed Captions on with subtitles fallback ("subtitles showing")</h3>
+<media-captions-button 
+  use-subtitles-fallback 
```

Comment:
➕ I went back and forth on this, and ultimately I agree to make `use-subtitles-fallback` the default.

Even though I like that the currently default is more technically & semantically correct, in a practical sense I think it hurts us more than it helps.

As a matter of fact, I tripped up on this with Demuxed because the Demuxed captions track on the live stream was being incorrectly marked as subtitles (despite the fact that the manifest had the attributes that should have presented them as captions ). I need to follow up on that, actually. Not sure if it was an Hls.js bug or that the manifest was incorrect.

So yes, from personal experience if `use-subtitles-fallback` was the default it would have been easier for me and I imagine other people would get tripped up by that too.